pkg/operator/sync_apiservice_v4_00.go

```diff
@@ -44,6 +44,18 @@ func syncAPIServiceController_v4_00_to_latest(c serviceCAOperator, operatorConfi
 		return fmt.Errorf("%q: %v", "clusterrolebinding", err)
 	}
 
+	requiredRole := resourceread.ReadRoleV1OrDie(v4_00_assets.MustAsset("v4.0.0/apiservice-cabundle-controller/role.yaml"))
```

Comment:
should be able to do this once instead of three times with the shared role and binding